### DISCUSSION TITLE
Dropdown of Select component does not close onClick outside in prod

### DISCUSSION THREAD

**Discussion by Igor-Arefev** [mantinedev/mantine#7068] [Q&A]
I have encountered a very strange behavior. I am using Select component and in local development it behaves predictably, when dropdown is open and I clik outside it closes. But in production, it doesn't work. And you have to either click on the dropdown itself or on the select element to close it. 

I use it inside the Paper component. I have Next JS 14, and library version ^7.13.2, also tailwind for styling

**Comment by rtivital** [comment]
Update Mantine packages to the latest version, the issue has been fixed.

**Reply by Igor-Arefev** [reply]
Thanks. Now it works

**Reply by atuls45** [reply]
I'm using "@mantine/core": "^8.1.2",
Still it is not working @rtivital
